pages/api/submit.js

```diff
@@ -41,11 +41,12 @@ const callLambda = async (form, fields) => {
   const lambdaClient = new LambdaClient({ region: "ca-central-1" });
 
   const command = new InvokeCommand({
-    FunctionName: process.env.SUBMISSION_API ?? "",
+    FunctionName: process.env.SUBMISSION_API ?? "Submission",
     Payload: JSON.stringify({
       form,
       responses: fields,
       submission,
+      vault: true,
```

Comment:
Not related to this PR, but eventually will we want this to be an option? As in, folks could use either email or vault (or any other) submission method we support?

Reply:
Exactly!

Reply:
Yup, I think I'll add a minimal "feature flag" functionality to our app, where we can turn features of/off via `.env` variables. This is a good candidate, and showing/hiding the "Login" link in the header is another :-)

Reply:
Just a reminder that env vars add complexity.  For every env var change there are 3-4 files in terraform that need to be changed and redeployed.

Reply:
On Portal we used a package that used a DB table to store flags that were controlled through Admin Page access.
Really help cut down on the number of infrastructure changes needed as new features were added.